wagtail/admin/templates/wagtailadmin/admin_base.html

```diff
@@ -29,6 +29,7 @@
     <script src="{% versioned_static 'wagtailadmin/js/telepath/telepath.js' %}"></script>
     <script src="{% versioned_static 'wagtailadmin/js/sidebar.js' %}"></script>
     <script src="{% versioned_static 'wagtailadmin/js/modal-workflow.js' %}"></script>
+    <script src="{% versioned_static 'wagtailadmin/js/privacy-switch.js' %}"></script>
```

Comment:
To test that we correctly include this, it should be simpler compared to the comments as we just want to make sure it's there on the views we are showing the Privacy Switch (similar to the Privacy Indicator but allows switching).

I would recommend `TestPrivacyIndicators` within the file `wagtail/admin/tests/test_privacy.py`.

Observe there's a range of tests, not all of them need the switch JS.

We should add the assertion in the test `test_edit_public`  & `test_edit_private` (as in, editing a page, the switch should be available).

The other place the switch is available is within the collection views, see `wagtail/admin/tests/test_collections_views.py` and the class `TestSetCollectionPrivacy`. Adding a single assertion to the test `test_get_private_child` should be suitable.